runtime/jvmti/jvmtiHelpers.c

```diff
@@ -1791,21 +1791,39 @@ static UDATA
 findDecompileInfoFrameIterator(J9VMThread *currentThread, J9StackWalkState *walkState)
 {
 	UDATA rc = J9_STACKWALK_KEEP_ITERATING;
+	J9Method *method = walkState->method;
+
+#if JAVA_SPEC_VERSION >= 20
+	if (NULL != method) {
```

Comment:
This can never be NULL due the visible only walk flag.

Reply:
Removed the `if` statement and converted it into an `assert`.